### ISSUE TITLE
Link to motion.txt?

### ISSUE THREAD

**Issue by snoblenet** [lifepillar/vim-cheat40#1]
The main cheat file says:
```
See `motion.txt`
```

This makes it look like a link to a file with that name.

But when you `gf` on the supposed link, Vim says: `E447: Can't find the file "motion.txt" in the path`, and indeed I can't find any `motion.txt` in this repository.

Do you mind explaining what I've missed here please?

Many thanks,

Steven

**Comment by lifepillar** [comment]
That is not a link, in fact, and you're right that the syntax coloring is misleading. I'll try to make it into a functional link.

**Comment by lifepillar** [comment]
Can you try the current master? You should be able to jump to `motion.txt` using `gf`, `<c-w><c-f>`, `<c-]>`, etc…. Similarly, for the other tags.

**Comment by lifepillar** [comment]
No, sorry, tags do not work. But you may still jump to the `*.txt` files with `gf` (although `<c-w>gf` is probably better because it opens a new tab).

**Comment by lifepillar** [comment]
Ok, now everything should work as expected: you may jump to files with both `gf` and tag commands. Please let me know if it works for you.

**Comment by snoblenet** [comment]
will do!

**Comment by snoblenet** [comment]
Hi @lifepillar. Thanks for creating Cheat 40. I've used it as a basis of my own personal cheat sheet system that I'll probably never post to GitHub as it contains numerous odd features that are specific to me and my learning journey -- e.g. I'm widening the window for longer explanations, highlighting alternating rows to visually link each shortcut with its explanation, including multi-step tips (e.g. project-wide search with args), and including some non-vim stuff I like to keep handy (e.g. for Git, Bash, etc).

I reload, updated and re-enabled Cheat 40. The syntax highlighting that I previously saw was no longer there but I do not know if that is because of changes to Cheat 40 or changes to my vimrc, which I've also radically changed in the mean time.

`motion.txt` was no longer highlighted as previously, and I could now see the full source for generating the full path in the help window. When my cursor was on `motion.txt` and I pressed `gf`, the target file opened in the cheat window.

I'm thinking I might borrow this part of your syntax to create multiple linked help files for myself, so something like `:cheat` while take me to the table of contents, while `:cheat-git` would take me straight to my cheat sheet for Fugitive, Vimdiff and Git in general. You've really got my mind whirring.

Thanks again.
